### ISSUE TITLE
client_addr  conflits consul reload command

### ISSUE THREAD

**Issue by appoule** [hashicorp/consul#5709]
When filing a bug, please include the following headings if possible. Any example text in this template can be deleted.

#### Overview of the Issue
consul reload failed to:
Error reloading: Put http://127.0.0.1:8500/v1/agent/reload: dial tcp 127.0.0.1:8500: connect: connection refused

A paragraph or two about the issue you're experiencing.

#### Reproduction Steps
According to the deployment guide @https://learn.hashicorp.com/consul/advanced/day-1-operations/deployment-guide
 1.  Consul has been started.
 2.During consul running, set client_addr = "hostip", ui = true in 
 /etc/consul.d/consul.hcl file
 doing is to have a chance access the UI
3. After step 2, run consul reload, issue seen.

 

### Consul info for both Client and Server

<details>
  <summary>Client info</summary>

```
output from client 'consul info' command here
```

</details>

<details>
  <summary>Server info</summary>

```
output from server 'consul info' command here
```

</details>

### Operating system and Environment details

OS, Architecture, and any other information you can provide about the environment.

### Log Fragments

Include appropriate Client or Server log fragments. If the log is longer than a few dozen lines, please include the URL to the [gist](https://gist.github.com/) of the log instead of posting it in the issue. Use `-log-level=TRACE` on the client and server to capture the maximum log detail.

**Comment by i0rek** [comment]
Can you show me your consul configuration? It seems like the Consul HTTP API is not listening on `127.0.0.1:8500`. While you are at it, could you also tell me which version you are using and send your logs?

**Comment by pearkes** [comment]
Given we haven't heard anything based on our suggestions/questions above I'm going to close this issue, but I encourage you to comment and we can re-open it if you want to pick this up again.

Alternatively, if things have changed dramatically, feel free to create a new issue or PR.
